huaweicloud/services/acceptance/vpc/resource_huaweicloud_vpc_traffic_mirror_filter_rule_test.go

```diff
@@ -0,0 +1,223 @@
+					resource.TestCheckResourceAttr(resourceName, "protocol", "icmpv6"),
+					resource.TestCheckResourceAttr(resourceName, "ethertype", "IPv6"),
+					resource.TestCheckResourceAttr(resourceName, "source_cidr_block", "2002:50::44/128"),
+					resource.TestCheckResourceAttr(resourceName, "source_port_range", ""),
+					resource.TestCheckResourceAttr(resourceName, "destination_cidr_block", "::/0"),
+					resource.TestCheckResourceAttr(resourceName, "destination_port_range", ""),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccTrafficMirrorFilterRule_egress(t *testing.T) {
+	var (
+		trafficMirrorFilterRule interface{}
+		name                    = acceptance.RandomAccResourceNameWithDash()
+		resourceName            = "huaweicloud_vpc_traffic_mirror_filter_rule.test"
+		rc                      = acceptance.InitResourceCheck(
+			resourceName,
+			&trafficMirrorFilterRule,
+			getTrafficMirrorFilterRuleResourceFunc,
+		)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccTrafficMirrorFilterRule_egress(name),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(resourceName, "traffic_mirror_filter_id",
+						"huaweicloud_vpc_traffic_mirror_filter.test", "id"),
+					resource.TestCheckResourceAttr(resourceName, "direction", "egress"),
+					resource.TestCheckResourceAttr(resourceName, "protocol", "all"),
+					resource.TestCheckResourceAttr(resourceName, "ethertype", "IPv4"),
+					resource.TestCheckResourceAttr(resourceName, "source_cidr_block", "192.168.1.0/24"),
+					resource.TestCheckResourceAttr(resourceName, "action", "accept"),
+					resource.TestCheckResourceAttr(resourceName, "priority", fmt.Sprintf("%v", 20)),
+					resource.TestCheckResourceAttr(resourceName, "source_port_range", ""),
+					resource.TestCheckResourceAttr(resourceName, "source_port_range", ""),
+					resource.TestCheckResourceAttr(resourceName, "destination_cidr_block", ""),
+					resource.TestCheckResourceAttr(resourceName, "description", ""),
+				),
+			},
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccTrafficMirrorFilterRule_step1(name string) string {
+	return fmt.Sprintf(`
+%s
+
+resource "huaweicloud_vpc_traffic_mirror_filter_rule" "test" {
+	traffic_mirror_filter_id = huaweicloud_vpc_traffic_mirror_filter.test.id
+	ethertype                = "IPv4"
+	direction                = "ingress"
+	protocol                 = "tcp"
+	action                   = "accept"
+	priority                 = 1
+	description              = "create VPC traffic mirror filter rule"
```

Comment:
invalid white space format, want two spaces, not tab

Reply:
fixed